packages/stream-model-instance-handler/src/model-instance-document-handler.ts

```diff
@@ -84,12 +84,12 @@ export class ModelInstanceDocumentHandler implements StreamHandler<ModelInstance
     const isSigned = StreamUtils.isSignedCommitData(commitData)
     if (isSigned) {
       const streamId = await StreamID.fromGenesis('MID', commitData.commit)
-      const { controllers, family } = payload.header
+      const { controllers } = payload.header
       await SignatureUtils.verifyCommitSignature(
         commitData,
         context.did,
         controllers[0],
-        family,
+        null,
```

Comment:
@stbrody I've added a null value right now to side-step the `family` requirement for `SignatureUtils.verifyCommitSignature()` but would like to follow up on it if this needs a breakout/duplication of this class for MID.

Reply:
We should change this method signature to accept `model` as `StreamID` instead of family. We should also make this param optional.

Reply:
Do we want that as part of this PR (removal) or with the follow up one as @stbrody you mentioned in a different comment as well.

Reply:
fine to leave for a follow-up IMO